src/utils/rules-proxy.js

```diff
@@ -80,7 +80,7 @@ const createRewriter = async function ({ configPath, distDir, jwtRoleClaim, jwtS
     const cookieValues = cookie.parse(req.headers.cookie || '')
     const headers = {
       'x-language': cookieValues.nf_lang || getLanguage(req.headers),
-      'x-country': cookieValues.nf_country || getCountry(),
+      'x-country': cookieValues.nf_country || getCountry(geoCountry),
```

Comment:
Can we just simplify that instead of having a function for that?

```suggestion
      'x-country': cookieValues.nf_country || geoCountry || 'us',
```

Reply:
@lukasholzer This is done.